cmake/Modules/ArmConfig.cmake

```
@@ -32,7 +34,7 @@ macro(get_arm_fpu result target)
         set(${result} vfp)
     elseif(target STREQUAL "RasPi2")
         set(${result} neon-vfpv4)
-    elseif(target MATCHES "RasPi3|RasPi4")
+    elseif(target MATCHES "RasPi3|RasPi4|RasPi5")
```

At this point we should use a range (same below):
```suggestion
    elseif(target MATCHES "RasPi[3-5]")
```